src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import classNames from "classnames";
+export const Tabs = ({tabs, activeTabId, onTabSelected, setOnTabSelected, setActiveTabId}) => {
```

Comment:
The `onTabSelected` prop should be a callback function, not a state object. This component should call `onTabSelected` with the `id` of the selected tab when a tab is clicked.